practice_2/WeatherApp/VIew/SettingsViewController.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+import Foundation
+
+enum SegmentIndex: Int {
+    case metric = 0
+    case imperial = 1
+}
+
+class SettingsViewController: UIViewController {
+
+    @IBOutlet weak var unitSegment: UISegmentedControl!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setInitialSegmentIndex()
+    }
+
+    private func setInitialSegmentIndex() {
+        if let unit = UserDefaults.standard.string(forKey: "temperutureUnit") {
```

Comment:
better to move key to some constant
using plain strings is error prone potentially